tests/tests_firewall_fact.yml

```diff
@@ -78,6 +78,7 @@
 
       always:
         - name: Cleanup
+          tags: tests::cleanup
```

Comment:
this should a list - so
```yaml
           tags:
             - tests::cleanup
```

Reply:
Oh, my mistake. This has been fixed now on both PRs.